app/web/features/badges/locales/en.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I'm not sure how we should go about localizing these. I guess doing this is not a bad idea. Then if we come up with a better way, we can always copy these translations over!